workspace.json

```diff
@@ -287,11 +281,9 @@
                     }
                 },
                 "serve": {
-                    "executor": "@nrwl/node:node",
+                    "executor": "@nrwl/vite:dev-server",
```

Comment:
I'm not sure if vite:dev-server the right executor for the backend server? I thought vite was primarily for frontend apps, but I could be wrong. I've only used it as a replacement for create-react-app

Reply:
Ah, I think you're right.

I'll check if we can leverage another executor instead. Might be why I'm having issues.